xUnique.py

```diff
@@ -217,21 +229,12 @@ def sort_pbxproj(self, sort_pbx_by_file_name=False):
         pbx_flag = False
         last_two = deque([])
 
-        def file_dir_cmp(x, y):
-            if '.' in x:
-                if '.' in y:
-                    return cmp(x, y)
-                else:
-                    return 1
-            else:
-                if '.' in y:
-                    return -1
-                else:
-                    return cmp(x, y)
+        def file_dir_order(x):
+            return ('.' in x, x)
```

Comment:
this new compare method does not act as what I did before. It will put all dirs and files in alphabet order. 
What I did before was putting dirs before files, and they both in alphbet order in their type. Doing a diff between the results would be more clear.
I will revert this change.